index.html

```diff
@@ -1,247 +1,368 @@
+                <h3>Profesional Art Printing Data</h3>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="Description-tags">
+                  <li>html</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button class="btn-desk btn btn-card" type="button">
+                See Project
+              </button>
+            </li>
+            <li class="project-card">
+              <div class="Card-description">
+                <h3>Profesional Art Printing Data</h3>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="Description-tags">
+                  <li>html</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button class="btn-desk btn btn-card" type="button">
+                See Project
+              </button>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <div id="About-and-skill">
+        <section id="about-section">
+          <div class="Header">
+            <h3 id="About-me">About me</h3>
+          </div>
+          <p class="About-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button type="button" class="btn btn-About-me">Get My Resume</button>
+        </section>
+        <div id="divider"></div>
+        <section id="skills">
+          <div class="tabs">
+            <ul class="centro">
+             <li> <span id="cube" class="recto">Language</span> </li>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+            <ul class="centro">
+             <li> <span id="brecto" class="recto">Frameworks</span> </li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CarpyBara</li>
+              <li>Selenium</li>
+            </ul>
+            <ul class="centro">
+             <li> <span id="circle" class="recto">Skills</span> </li>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </section>
+      </div>
+      <div class="footer">
+        <section id="contact-form" class="work-section">
```

Comment:
- [ ] Make sure the contact section looks like this
- Your Design
![Capturec](https://user-images.githubusercontent.com/81444338/171842023-25d6f436-c2c7-429f-89a0-db07e0d2d077.PNG)
Figma Design
![Capturefc](https://user-images.githubusercontent.com/81444338/171842160-a60d94bf-0afc-4a62-8eeb-0f08e73801f6.PNG)